src/constants/chains.tsx

```diff
@@ -39,7 +41,6 @@ export type ChainConfig = {
   }
   isL2?: boolean
   iconUrls: string[]
-  id: ChainsValues
```

Comment:
Good catch, it's a repeated field.

Can we double check that we don't use it elsewhere in the dapp?

Reply:
Sure, I fixed all ts errors